likecoin/views/metabox.php

```diff
@@ -1,35 +1,61 @@
 <?php
   function likecoin_add_meta_box($post) {
     $author = $post->post_author;
-    $likecoin_id = get_author_likecoin_id($post);
+    $likecoin_id = get_user_meta($author, 'lc_likecoin_id', true);
+    $likecoin_wallet = get_user_meta($author, 'lc_likecoin_wallet', true);
+    $widget_position = get_post_meta($post->ID, 'lc_widget_position', true);
+    if (!$widget_position) $widget_position = get_user_meta($author, 'lc_widget_position', true);
+    if (!$widget_position) $widget_position = '';
```

Comment:
Default value is 'none'?

Reply:
`none` means no default option is defined for `<select>`, then naturally the first item `Top and bottom` would be default